src/chains/opStack/actions/getGame.ts

```diff
@@ -0,0 +1,99 @@
+import type { Client } from '../../../clients/createClient.js'
+import type { Transport } from '../../../clients/transports/createTransport.js'
+import type { ErrorType } from '../../../errors/utils.js'
+import type { Account } from '../../../types/account.js'
+import type {
+  Chain,
+  DeriveChain,
+  GetChainParameter,
+} from '../../../types/chain.js'
+import {
+  GameNotFoundError,
+  type GameNotFoundErrorType,
+} from '../errors/withdrawal.js'
+import type { GetContractAddressParameter } from '../types/contract.js'
+import type { Game } from '../types/withdrawal.js'
+import { type GetGamesErrorType, getGames } from './getGames.js'
+
+export type GetGameParameters<
+  chain extends Chain | undefined = Chain | undefined,
+  chainOverride extends Chain | undefined = Chain | undefined,
+  _derivedChain extends Chain | undefined = DeriveChain<chain, chainOverride>,
+> = GetChainParameter<chain, chainOverride> &
+  GetContractAddressParameter<
+    _derivedChain,
+    'portal' | 'disputeGameFactory'
+  > & {
+    /**
+     * The minimum block number of the dispute game.
+     */
+    l2BlockNumber: bigint
+    /**
+     * Limit of games to extract.
+     * @default 100
+     */
+    limit?: number | undefined
+    /**
+     * Strategy for extracting a dispute game.
+     *
+     * - `latest` - Returns the latest dispute game.
+     * - `random` - Returns a random dispute game.
```

Comment:
```suggestion
     * - `random` - Returns a random dispute game. This helps prevent DDOS from someone submitting invalid games
```
Not obvious to most people "why"?